test/functorch/test_control_flow.py

```diff
@@ -2358,7 +2487,8 @@ def test_cond_vmap(self, predType, innerFnType, nOperands, nClosure, nesting):
 
     def test_cond_vmap_simple(self):
         def fn(x):
-            return torch.cond(
+            # return torch.cond(
```

Comment:
By removing the circular imports, we should be able to leave these unchanged.

Reply:
Indeed this resolved the issue